### ISSUE TITLE
Sentry stopped sending messages in Slack after upgrade

### ISSUE THREAD

**Issue by nullday0** [getsentry/onpremise#736]
## Version Information

Version: Sentry 20.10.17 11e61c

## Description

I've decided to upgrade my slack via running `./install.sh`, everything went smooth and works fine.
Except Slack integration. After the upgrade it doesn't send messages anymore.
I've checked Alerts page, it's all like it was before, I've tried to edit several rules and save it, it didn't help.
I went to Settings and checked Slack integration. I've tried to remove my workspace and add it again, it didn't help.

Maybe there is some cache that stuck or something like that? Logs of what container should I check for errors?
I've tried to install :nightly build as well, it didn't change anything.

During install I saw this: 
```
sentry/sentry.conf.py already exists, skipped creation.
sentry/config.yml already exists, skipped creation.
sentry/requirements.txt already exists, skipped creation.
symbolicator/config.yml already exists, skipped creation.
relay/config.yml already exists, skipped creation.

Fetching and updating Docker images...

Some service image(s) must be built from source by running:
    docker-compose build post-process-forwarder sentry-cleanup ingest-consumer web worker snuba-cleanup cron symbolicator-cleanup
```

After install I've run `docker-compose build post-process-forwarder sentry-cleanup ingest-consumer web worker snuba-cleanup cron symbolicator-cleanup` and then restarted all containers. Nothing helped.

Thanks.

**Comment by BYK** [comment]
Hi,

Have you checked https://develop.sentry.dev/integrations/slack/ yet?

**Comment by nullday0** [comment]
I was able to figure out what happened.
I've run `docker logs onpremise_post-process-forwarder_1` and saw errors like this: 
```
Exception: KafkaError{code=OFFSET_OUT_OF_RANGE,val=1,str="Broker: Offset out of range"}
```

Then I've found this issue: https://github.com/getsentry/onpremise/issues/478
I've followed the guide from the issue: https://github.com/getsentry/onpremise/issues/478#issuecomment-666254392
And it worked all well.

Thanks.

P.S. The reason for that issue I guess was that my server was hard rebooted before that.
